wire-ios/Wire-iOS/Sources/Helpers/String+Capitalization.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension String {
+    /// The first character in each sentence changed to its corresponding uppercase value.
+    /// All remaining characters set to their corresponding lowercase values.
+    ///
+    /// Caveat:
+
+    ///  1. The implementation only works with a string that contains one sentence.
+    ///  The second sentence won't get capitalized.
+    ///
+    ///  2. It doesn't work if the first string is a whitespace character and delimiter.
+    var capitalizedCharacter: String {
+        let firstLetter = self.prefix(1).capitalized
+
+        let remainingLetters = self.dropFirst().lowercased()
+
+        return firstLetter + remainingLetters
```

Comment:
where we're going to use it? I'm asking because that might be a problem if there's more than one sentence. In that case second sentence starts with lowercase

Reply:
We're going to use it on buttons, placeholders for text fields as well as labels. In most if not all cases, we don't have to deal with more than one sentence.

Reply:
Are you sure we don't want to be using localized capitalization? Each locale may have different ways to capitalize... it seems like we should adhere to localization rather than forcing a specific style, or is that intended?

Reply:
@johnxnguyen I talked with Astrid and she mentioned that the first letter needs to be capitalized no matter the locale. So we can go ahead with that implementation

Reply:
@agisilaos  I think Astrid misunderstood you. Nouns in German should always be capitalized:
"Nach **P**ersonen und **G**ruppen suchen".
 I think we should capitalise only first word.

Reply:
The name capitalizedCharacter doesn't tell us which character or that it lowercases the others. I suggest capitalizingFirstCharacterOnly

Reply:
Done!